pkg/operation/seed/seed.go

```diff
@@ -535,7 +535,12 @@ func BootstrapCluster(k8sGardenClient kubernetes.Interface, seed *Seed, config *
 			"enabled": loggingEnabled,
 			"ingress": map[string]interface{}{
 				"basicAuthSecret": basicAuth,
-				"host":            kibanaHost,
+				"hosts": []map[string]interface{}{
+					{
+						"hostName":   kibanaHost,
+						"secretName": "kibana-tls",
```

Comment:
Why don't we use the wildcard cert here as well?

Reply:
Good idea. I've changed the ingress hostnames for Kibana, Grafana and Prometheus in the `garden` namespace immediately.